docs/blog/2019-08-30-speed-up-your-time-to-interactive-by-delaying-third-party-scripts/index.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Speed up your time to interactive by delaying third party scripts
+date: 2019-08-30
+author: Kilian Valkhof
+excerpt: "Delay loading scripts to lower your time to interactive by over 60%"
+tags:
+  - performance
+  - seo
+---
+
+Gatsby does many things to get a website visible as fast as possible. But getting a website visible isn't the only interesting metric. There are others that, if you're not careful, can easily take much longer than needed. Even if your site still feels fast.
+
+The one this article focuses on is "[Time to interactive](https://developers.google.com/web/tools/lighthouse/audits/time-to-interactive)", which is the time it takes for all your initial scripts to run. This one is particularly interesting for a couple of reasons:
+
+- It's not super noticable when you use a website on desktop
+- Scripts loaded asynchronously are counted towards this number (even though they're async, they're still loaded in sequence after a page load)
+- It's a very important part of [Google's performance ranking](https://github.com/GoogleChrome/lighthouse/blob/master/docs/scoring.md). Because of the user impact, it's 5 times more important than the "[first meaningful paint](https://developers.google.com/web/tools/lighthouse/audits/first-meaningful-paint)", e.g. when something shows up on the screen. **5 times!**
+
+These together might mean that you have a website that _feels_ fast under some circumstances, but Google will give you a penalty because it's slow.
+
+To get an overview of how your site scores, you can use Lighthouse, either in your Chrome developer tools by going to the "Audits" tab, or by going to [web.dev](https://web.dev/measure). This will test your site on a number of different metrics, such as accessibility, SEO (Search Engine Optimization), and best practices, but the one we're interested in is "performance".
```

Comment:
there is also https://developers.google.com/speed/pagespeed/insights/ if you just want performance or the chrome extension if you want the latest and greatest lighthouse. Audit panel is mostly on an outdated version.

I don't think it's necessary to point this out but just stating it in the review as other possible values.

Reply:
Yeah there's the CLI too etc. I think pointing people to web.dev is the best method but that's not available for all sites (localhosts, intranet pages), then the audit panel is better.